app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -30,6 +30,11 @@ open class BrowserApplication : Application() {
     override fun onCreate() {
         super.onCreate()
 
+        // Set up Glean as early as possible in order to ensure availability to the crash reporting
+        // service.  Glean provides its own assurances that it is running on the main process so it
+        // should be safe calling the following function from another process.
+        setupGlean(this)
```

Comment:
This looks good!

Unrelated: since you're touching this, would you mind moving [this line](https://github.com/mozilla-mobile/reference-browser/blob/1f008487d375e7f578da4497b9a04561cb160a9e/app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt#L78) before `Glean.initialize()`?